server/graphql/v2/input/ApplicationCreateInput.js

```diff
@@ -0,0 +1,13 @@
+import { GraphQLInputObjectType, GraphQLString } from 'graphql';
+
+import URL from '../scalar/URL';
+
+export const ApplicationCreateInput = new GraphQLInputObjectType({
+  name: 'ApplicationCreateInput',
+  description: 'Input type for Application',
+  fields: () => ({
+    name: { type: GraphQLString },
+    description: { type: GraphQLString },
+    callbackUrl: { type: URL },
```

Comment:
Should we add the type here?

Reply:
What type?

Reply:
Application type (oauth)

Reply:
Even if it has only one value for now, to make it explicit that we're creating an oauth app, not an API key